### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by atharvabhanage02**
This is a short PR : 

- User can navigate to Single Video Page from Explore Page
- Has functionality of Must Watch Videos in Single Video Page 

Files to look for : 

- SingleVideoCard : `src/Components/SingleVideoCard`
- SingleVideoPage : `src/Pages/SingleVideoPage`

Deploy Preview : https://deploy-preview-3--cricplay.netlify.app/

### CODE REVIEW

File: src/Components/VideoMenu/VideoMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  MdOutlineWatchLater,
+  CgPlayList,
+  CgPlayListCheck,
+  BiLike,
+  AiFillLike,
+  RiHistoryLine,
+} from "../../Utils/icons";
+import { useLocation, useNavigate } from "react-router-dom";
+import { useAuth } from "../../Context/Auth/auth-context";
+import { useVideos } from "../../Context/VideosContext/VideosContext";
```
Comment (harshalini) [inline src/Components/VideoMenu/VideoMenu.jsx:12]: Suggestion: You can make a separate file for exporting context from there

File: src/Pages/SingleVideoPage/SingleVideoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { Navbar } from "../../Components/Navbar/Navbar";
+import { Sidebar } from "../../Components/Sidebar/Sidebar";
+import { SingleVideoCard } from "../../Components/SingleVideoCard/SingleVideoCard";
```
Comment (harshalini) [inline src/Pages/SingleVideoPage/SingleVideoPage.jsx:5]: Same here

**Review by harshalini — APPROVED** [review]
PR is good to merge

**Review by mansipadgelwar — APPROVED** [review]
Looks good.

File: src/Components/VideoMenu/VideoMenu.jsx (hunk 1 above)
Comment (atharvabhanage02) [reply src/Components/VideoMenu/VideoMenu.jsx:12]: Yeah , I'll do that

**Pull request merged**